leshan-client-core/src/main/java/org/eclipse/leshan/client/object/OSCORESecurity.java

```diff
@@ -0,0 +1,109 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Sierra Wireless and others.
+ * 
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v1.0 which accompany this distribution.
+ * 
+ * The Eclipse Public License is available at
+ *    http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ *    http://www.eclipse.org/org/documents/edl-v10.html.
+ * 
+ * Contributors:
+ *     Sierra Wireless - initial API and implementation
+ *     Rikard Höglund (RISE SICS)
+ *
+ *******************************************************************************/
+package org.eclipse.leshan.client.object;
+
+import org.eclipse.leshan.client.request.ServerIdentity;
+import org.eclipse.leshan.client.resource.BaseInstanceEnabler;
+import org.eclipse.leshan.client.resource.LwM2mInstanceEnabler;
+import org.eclipse.leshan.core.node.LwM2mResource;
+import org.eclipse.leshan.core.response.ReadResponse;
+import org.eclipse.leshan.core.response.WriteResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * A simple {@link LwM2mInstanceEnabler} for the OSCORE Security (0) object.
+ */
+public class OSCORESecurity extends BaseInstanceEnabler {
+
+    private static final Logger LOG = LoggerFactory.getLogger(Security.class);
+
+    byte[] OSCORE_master_secret;
+    String OSCORE_sender_id;
+    String OSCORE_recipient_id;
+    int OSCORE_AEAD_algorithm;
+    int OSCORE_HMAC_algorithm;
```

Comment:
1. Generally in Java, camelCase is prefered, so we do that in Leshan too.
One exception in Leshan, we are using snake case for test name.
So it should looks like `oscoreMasterSecret`.

2. Any reason do not use private access modifier ?

3. I know in the LWM2M specification,  resource name are prefixed by OSCORE and generally I prefer to keep as closest as possible as specification naming. But in this case I think it should be less verbose to just remove the `OSCORE` part in attribute name. So it looks like `masterSecret`, does it sound good to you ?